cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -275,8 +277,17 @@ class EvaluatorCommand : OrtCommand(
         }
 
         if (packageCurationsDir != null || packageCurationsFile != null) {
-            val provider = FilePackageCurationProvider(packageCurationsFile, packageCurationsDir)
-            ortResultInput = ortResultInput.replacePackageCurations(provider, providerId = "EvaluatorCommandOption")
+            val packageCurationProviders = buildList {
+                if (ortConfig.enableRepositoryPackageCurations) {
```

Comment:
This should IMO use logic analog to the SetCommand touched in this PR as well, rather than relying on `enableRepositoryPackageCurations`.

Reply:
To me the previous behavior makes sense for a helper command like `SetCommand`, but for the `EvaluatorCommand` it makes more sense to behave according to the global configuration.

Reply:
I preferred not to consume that config property in the evaluator, to not start worrying about config changes inbetween the stages. 

Anyhow, I believe we should talk about the overall picture how resolving configuration is supposed to work, then such details will be clear anyway. I'm missing this clarity / alignment a bit which makes it hard for me to review this.

Reply:
> I preferred not to consume that config property in the evaluator, to not start worrying about config changes inbetween the stages.

The command already reacts to global configuration changes in the rules, license classifications, rule violation threshold, and so on. It would be rather inconsistent if this was the one config property it ignores.

> Anyhow, I believe we should talk about the overall picture how resolving configuration is supposed to work, then such details will be clear anyway. I'm missing this clarity / alignment a bit which makes it hard for me to review this.

The next PR will make the idea a lot more clear, I would suggest we have the discussion there as this PR only has a minor functional change in the `EvaluatorCommand`.

Reply:
> Not sure what your points is, by default all configuration files are located in ~/.ort/config/ 

My point is to distinguish between:
 
1. the tooling config
2. entries in some upstream database like clearly defined, OSV or `ort-config`

> That's why I'm currently working on increasing the consistency while at the same time not breaking the orthw workflow.

Maybe again: propose to have a chat about how the config resolution shall work in the end and how the mentioned `orthw` workflow may actually look like in future.